pennylane/operation.py

```diff
@@ -116,7 +116,7 @@
 import autograd.numpy as np
 
 from .qnode import QNode, QuantumFunctionError
-from .utils import _flatten, _unflatten
+from pennylane.utils import _flatten, _unflatten
```

Comment:
Curious about this line change as well - was there a circular import issue?

Reply:
Same here, I did not remove this line consciously, must have been during a pull? Added it to the file.

Reply:
Again, not sure why this sneaked in, I did not change it deliberately. Reinstated the old version.